js/script.js

```diff
@@ -113,39 +117,76 @@ const createTaskCard = (task) => {
   } else {
     const titleElement = document.createElement("p");
     titleElement.textContent = task.title;
+    titleElement.classList.toggle("line-through", task.isCompleted);
     taskCard.appendChild(titleElement);
 
     const createdAtElement = document.createElement("p");
     createdAtElement.className = "created-at";
     createdAtElement.textContent = `Created At: ${task.createdAt}`;
     taskCard.appendChild(createdAtElement);
 
+    const actionsContainer = document.createElement("div");
+    actionsContainer.className = "task-actions";
+
+    const buttonsContainer = document.createElement("div");
+    buttonsContainer.className = "task-buttons";
+
+    const checkButton = document.createElement("button");
+    checkButton.className = "btn-check";
+    checkButton.innerHTML = `
+      <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M20.285 2L9 13.567L3.714 8.556L0 12.272L9 21L24 5.715L20.285 2Z" fill="#BBBDD0"/>
+      </svg>
+    `;
```

Comment:
declare the `svg` files in another `constants.js` file and export it from that. Then by importing it, use it in `script.js `file.